modules/arm_plugin/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -46,6 +46,10 @@ std::vector<std::string> disabledTestPatterns() {
 #ifdef __arm__
         // Sporadic hanges on linux-debian_9_arm runner (armv7l) 72140
         ".*canStartSeveralAsyncInsideCompletionCallbackWithSafeDtor.*AUTO.*",
+#endif
+#ifdef __aarch64__
+        // Bus error on aarch64, 94254
+        ".*THROUGHPUT.*",
```

Comment:
If there is only 1 failed test, isn't it too wide regexp?
Probably, `.*canSetInputPrecisionForNetwork.*`?

Reply:
Let's, first of all, enabling something. Then, we can use infra and enable tests back.
I suggested to use `THROUGHPUT` because I suppose it can be connected with THROUGHPUT perf mode

Reply:
Right, we can use that next tput mode test failed:
```

[2022-10-14T17:41:59.392Z] [2022-10-14 19:41:59,972] [547664310720] armFuncTests-0 INFO: [ RUN      ] smoke_BehaviorTests/ExecNetSetPrecision.canSetInputPrecisionForNetwork/netPRC=FP16_targetDevice=CPU_config=(CPU_THROUGHPUT_STREAMS=CPU_THROUGHPUT_AUTO_)

[2022-10-14T17:41:59.392Z] [2022-10-14 19:41:59,982] [547664310720] armFuncTests-0 INFO: Segmentation fault

[2022-10-14T17:41:59.392Z] [2022-10-14 19:41:59,983] [547664310720] armFuncTests-0 INFO: ========== Completed. Exit code: 139, elapsed time: 2.069 sec, 0:00:02

[2022-10-14T17:41:59.392Z] [2022-10-14 19:41:59,984] [547664310720] armFuncTests-0 INFO: 
```

Reply:
Why doesn't the 2nd THROUGHPUT test skipped if we skip all THROUGHPUT tests in the PR?

Reply:
because skip config file works for tests, where `SKIP_IF_CURRENT_TEST_IS_DISABLED` is inserted, but this is done not for all tests

Reply:
If it is OK, we can temporarily exclude it via gtest_filter - https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1482